README.rst

```diff
@@ -75,6 +75,22 @@ be used to check that the minimum required versions didn't change. The following
   script:
   - vermin -t=2.7 -t=3 project_package otherfile.py
 
+Vermin can also be used as a `pre-commit <https://pre-commit.com/>`__ hook:
+
+.. code-block:: yaml
+
+  repos:
+    - repo: https://github.com/netromdk/vermin
+      rev: e88bda9
+      hooks:
+        - id: vermin
+          # specify your target version here, OR in a Vermin config file as usual:
+          args: ['-t=3.8']
```

Comment:
Let's update this line to be the following in relation to the comments to the YAML file:
```suggestion
          args: ['-t=3.8-', '--violations']
```